compatibility_lib/compatibility_lib/testpkgs/added_func/0.1.0/main.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2018 Google LLC
```

Comment:
Probably we need to make more complicated test files to cover all the types we are comparing including modules, classes, functions etc...

Reply:
Yes I think more test cases are definitely needed, but as to not overwhelm the review process, maybe the complexity of test cases can be scaled up over different PRs?